app/serializers/registration-reqeust.ts

```diff
@@ -0,0 +1,10 @@
+import OsfSerializer from './osf-serializer';
```

Comment:
ahh, this file is misspelled as well

Reply:
Am I importing from the wrong place?

Reply:
I think it's a typo in the file name: `app/serializers/registration-reqeust.ts` => `app/serializers/registration-request.ts`